sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+|--------------------|---------------------------------------------------------------------------------------------------------|
+| SIP-10 Token       | A token on the Stacks blockchain that adheres to the fungible token standards outlined in [SIP-10](https://github.com/stacksgov/sips/blob/main/sips/sip-010/sip-010-fungible-token-standard.md).        |
+| sBTC               | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
+| sBTC operation     | An operation that initiates some action from the sBTC protocol.                                         |
+| .sbtc contract     | A smart contract (or a collection of contracts) defining the sBTC token and functions related to it     |
+| sBTC Peg Wallet    | The single UTXO holding the entire BTC balance that’s pegged into sBTC. This peg wallet is managed and maintained by the sBTC Signers. |
+| Stacks Signer      | An entity that receives PoX payouts for stacking their STX tokens and actively participating in the Stacks protocol by signing mined blocks. |
+| sBTC Signer        | An entity that will sign sBTC operations and communicate with contracts on the chain to make that feasible. This entity has partial access to spending the sBTC UTXO. In this release, the sBTC signer is wholly separate from the Stacks Signer |
+| sBTC Signer Set    | The set of all sBTC signers. Each is registered with the .sbtc contract and the transfer. These entities as a group have full democratic access to the sBTC UTXO. |
+| sBTC Signer API    | API exposed by the sBTC Signer that handles basic low level commands. Remains private except to the Deposit API. |
+| Deposit API        | A third party API that communicates with the sBTC Bootstrap Signers via the sBTC Signer API.             |
+
+## Problem Statement
+Bitcoin is limited in its programmability and scalability. While its security and censorship-resistance make it a compelling platform to build decentralized applications, transaction confirmation times do not meet the modern needs of users and developers. 
+
+- **Bitcoin’s Limited Programmability:** Bitcoin’s script has limited programmability, making it unsuitable for most decentralized applications. This leaves users with very few options to use financial applications, like lending and trading, without entrusting their Bitcoin to centralized entities. This also exposes users to counterparty risk, which has the potential to result in lost funds. 
+- **Bitcoin’s Slow Transaction Times:** Bitcoin is limited in its ability to process large amounts of data quickly and efficiently. Today, Bitcoin creates new blocks every 10 minutes on average, an interval which is longer than some alternative blockchains. This prohibits many types of applications that require faster confirmation times. 
```

Comment:
Can you give examples of applications that need faster confirmations?

Also, I think it's good enough to just say that Bitcoin's slow transaction times _and_ their slow confirmation times are a well-known and widely-felt source of user frustration, so _anything_ we can do to speed up transaction confirmation would be a win.